Extension/.eslintignore

```diff
@@ -2,3 +2,4 @@
 test/**/index.ts
 test/**/runTest.ts
 tools/prepublish.js
+vscode.d.ts

```

Comment:
What is vscode.d.ts doing in the root folder?  I thought it should only be in the node_modules folder.

Reply:
`"postinstall": "npx vscode-dts dev && npx vscode-dts main"` puts it in the root.

Reply:
From https://github.com/microsoft/vscode-dts#readme

Reply:
Looks like the node_modules location is the old one, i.e. they have code to delete that.
```typescript
  if (fs.existsSync('node_modules/vscode/vscode.d.ts')) {
    fs.unlinkSync('node_modules/vscode/vscode.d.ts')
```

Reply:
Sounds good.  It sounds like it should be in `.vscodeignore` too then.

Reply:
And use the same pattern as in the `.gitignore` so that both `.d.ts` files are excluded?

Reply:
Yeah, good catch, I didn't notice that.